be/src/common/config.h

```diff
@@ -707,6 +707,7 @@ CONF_Bool(enable_orc_late_materialization, "true");
 CONF_Int32(orc_file_cache_max_size, "2097152");
 // parquet reader, each column will reserve X bytes for read
 CONF_mInt32(parquet_buffer_stream_reserve_size, "1048576");
+CONF_mBool(parquet_enable_coalesce_reads, "true");
```

Comment:
I suggest parquet_coalesce_read_enable 
which is another format of parquet.coalesce_read.enable
Then other config about this feature can be `parquet.coalesce_read.xxx`

Reply:
OK.